src/fidgets/community/nouns-dao/components/ProposalDetailView.tsx

```diff
@@ -95,19 +131,25 @@ export const ProposalDetailView = ({
     return null;
   }
 
-  const proposer = proposal.proposer;
-  const sponsor = proposal.signers.length ? proposal.signers[0] : null;
-  const updates = proposal.status.updateMessages;
-  const version = 1 + updates.length;
-
   const votes = {
-    for: proposal.votes.filter((v) => v.support == "FOR").length,
-    nay: proposal.votes.filter((v) => v.support == "AGAINST").length,
-    abs: proposal.votes.filter((v) => v.support == "ABSTAIN").length,
-    total: proposal.votes.length,
-    quorum: proposal.quorumVotes,
+    for: Number(proposal.forVotes),
```

Comment:
We should be able to know the type of this ahead of time, right? Not have to cast it?

Reply:
The graphql api returns all values as a string

Reply:
I didn't create the full type for the response for now because it's rather lengthy